### PR TITLE
Test for add second restaurant

### PR DESCRIPTION
**Pull request by Zeicks**

### CODE REVIEW

File: src/main/java/com/easyrest/pages/OwnerPanel.java [hunk 1]
```diff
@@ -38,6 +34,18 @@ public OwnerPanel(WebDriver driver) {
         waiters = new Waiters(driver);
     }
 
+    private boolean isDisplayedElement(By element) {
+        return this.driver.findElement(element).isDisplayed();
+    }
+
+    public boolean isEmployeeDisplayed(String name) {
+        return isDisplayedElement(By.xpath("*//div//span[normalize-space()='" + name + "']"));
```
Comment (martadiachok) [inline src/main/java/com/easyrest/pages/OwnerPanel.java:42]: why did you write x path here and not where is variable?

Comment (Zeicks) [reply src/main/java/com/easyrest/pages/OwnerPanel.java:42]: I need to substitute a name from a method in xpath.
That's why I can not write is in variable.

File: src/test/java/com/easyrest/tests/owner/ActionWithEmployeeTest.java [hunk 2]
```diff
@@ -37,12 +39,13 @@ public void goToSignInOwner() {
     public void addWaiterTest() {
         createNewWaiterFacade = new CreateNewWaiterFacade(driver);
         createNewEmployee = new CreateNewEmployee(driver);
```
Comment (vadymtsehelny) [inline src/test/java/com/easyrest/tests/owner/ActionWithEmployeeTest.java]: just check if you use this object: createNewEmployee = new CreateNewEmployee(driver);

Comment (Zeicks) [reply src/test/java/com/easyrest/tests/owner/ActionWithEmployeeTest.java]: Thanks, cleaned

**Review by vadymtsehelny — APPROVED** [review]

**Review by dmatsev — APPROVED** [review]

**Review by martadiachok — APPROVED** [review]

**Review by ViktoriaMilkevich — APPROVED** [review]

**Pull request merged**